velox/substrait/tests/Substrait2VeloxPlanConversionTest.cpp

```diff
@@ -268,9 +268,9 @@ TEST_F(Substrait2VeloxPlanConversionTest, q6) {
 
   // Find and deserialize Substrait plan json file.
   std::string planPath =
-      getDataFilePath("velox/substrait/tests", "data/sub.json");
```

Comment:
@PenghuiJiao Is sub.json not used anymore? If so, let's delete it.

Reply:
Got it. Thank you for clarifying. I'm going to re-run tests and land if they pass. Next time, please, remember to update PR description when making additional changes.